libs/etterlatte-brev-model/src/main/kotlin/no/nav/etterlatte/brev/model/oms/EtteroppgjoerBrevData.kt

```diff
@@ -68,11 +69,27 @@ object EtteroppgjoerBrevData {
     ) : BrevVedleggInnholdData()
 
     data class Vedtak(
+        val vedleggInnhold: List<Slate.Element> = emptyList(),
         val bosattUtland: Boolean,
+        val etteroppgjoersAar: Int,
+        val avviksBeloep: Kroner,
+        val stoenadsBeloep: Kroner,
+        val resultatType: EtteroppgjoerResultatType,
+        val inntekt: Kroner,
+        val faktiskInntekt: Kroner,
+        val grunnlag: EtteroppgjoerBrevGrunnlag,
     ) : BrevFastInnholdData() {
         override val type: String = "OMS_EO_VEDTAK"
 
-        override fun medVedleggInnhold(innhold: () -> List<BrevInnholdVedlegg>): BrevFastInnholdData = this
+        override fun medVedleggInnhold(innhold: () -> List<BrevInnholdVedlegg>): BrevFastInnholdData =
+            this.copy(
+                vedleggInnhold =
+                    innhold()
+                        .single {
+                            it.key == BrevVedleggKey.OMS_EO_FORHAANDSVARSEL_BEREGNING
+                        }.payload!!
+                        .elements,
```

Comment:
```suggestion
                    krevIkkeNull(innhold()
                        .singleOrNull {
                            it.key == BrevVedleggKey.OMS_EO_FORHAANDSVARSEL_BEREGNING
                        }?.payload) {
                           "Mangler påkrevd vedlegg for etteroppgjør beregningsvedlegg"
                        }
                        .elements,
```

eller noe i den duren, så får vi mer leselige feil hvis vi ikke har vedlegget